src/operators/bufferCount.ts

```diff
@@ -33,7 +33,7 @@ class BufferCountSubscriber<T> extends Subscriber<T> {
     const count = (this.count += 1);
     const destination = this.destination;
     const bufferSize = this.bufferSize;
-    const startBufferEvery = this.startBufferEvery;
+    const startBufferEvery = (this.startBufferEvery == null) ? bufferSize : this.startBufferEvery;
```

Comment:
Some of other operator such as windowCount it has default value as 0 instead of null. Not sure if there's consistent recommendation or guidelines, so did not modify default value itself but simply take buffersize if value's not specified.